scripts/linters/pylint_extensions.py

```diff
@@ -540,11 +595,60 @@ def check_functiondef_params(self, node, node_doc):
-        are indented properly. Parameters should be indented by 4 relative to
-        the 'Arg: ' line and any wrap-around descriptions should be indented
-        by 8. The rest of the docstring section checks are in DocstringChecker.
+    def check_docstring_section_style(self, node):
+        """Checks whether the docstring has valid style.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or
+                method definition in the AST.
+        """
+        if node.name in self.constructor_names:
+            class_node = checker_utils.node_frame_class(node)
+            if class_node is not None:
+                self.check_docstring_structure(class_node)
+        self.check_docstring_structure(node)
+
+    def check_docstring_structure(self, node):
+        """Checks whether the docstring has the correct structure i.e.
+        do not have space the beginning and have a period at the end of
+        docstring.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or
+                method definition in the AST.
+        """
+        if node.doc:
+            docstring = node.doc.splitlines()
+            # Check for space after """ in docstring.
+            if docstring[0][0] == b' ':
+                self.add_message('space-after-triple-quote', node=node)
+            # Check if single line docstring span two lines.
+            elif len(docstring) == 2 and docstring[-1].strip() == b'':
+                self.add_message(
+                    'single-line-docstring-span-two-lines', node=node)
+            # Check for punctuation at end of a single line docstring.
+            elif (len(docstring) == 1 and docstring[-1][-1] not in
+                  ALLOWED_TERMINATING_PUNCTUATIONS):
+                self.add_message('no-period-used', node=node)
+            # Check for punctuation at the end of a multiline docstring.
+            elif len(docstring) > 1:
+                if docstring[-2].strip() == b'':
+                    self.add_message('empty-line-before-end', node=node)
+                elif docstring[-1].strip() != b'':
+                    self.add_message(
+                        'no-newline-used-at-end', node=node)
+                elif (docstring[-2][-1] not in
+                      ALLOWED_TERMINATING_PUNCTUATIONS and not
+                      any(word in docstring[-2] for word in EXCLUDED_PHRASES)):
+                    self.add_message('no-period-used', node=node)
+
+    def check_docstring_section_indentation(self, node):
+        """Checks whether the function argument definitions ("Args": section,
+        "Returns": section, "Yield": section, "Raises: section) are indented
+        properly. Parameters should be indented by 4 relative to the 'Args:'
+        'Return:', 'Raises:', 'Yield:' line and any wrap-around descriptions
+        should be indented by 8. The rest of the docstring section checks are
```

Comment:
Please remove the last line of the comment. There is no other class now. (The rest of the docstring section checks are in DocstringChecker.)

Reply:
Removed.